src/config/manifest.rs

```diff
@@ -38,4 +38,26 @@ impl CargoMetadata {
             manifest_path,
         })
     }
+
+    pub fn new_sync(manifest: &Path) -> Result<Self> {
```

Comment:
No way to reuse this code somehow? I admit I'm not an expert on Rust async.. but it would be nice.

Reply:
Well, we could just have a `new` function sync and make a new `new_async` fn which calls the `new` function. I'm not sure, unless there's a way to run async fns as sync? I don't know of any :(